src-v2/Not/Not/TimerUtility/TimerUtility.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Timers;
+
+namespace Not.TimerUtility;
+public class TimerUtility
```

Comment:
We're avoiding the `Utility` name by convention. Utilities or Helpers should be very general purpose objects, in our case we use Helpers and they are mostly static.

As it currently is, I don't see much benefit from this class, as it barely does anything, whilst it's more restrictive than the System.Timers.Timer, which is why I presume you've opted to simply expose it. It is sometimes a good idea to restrict an API simply in order to enforce a consistent usage, but you're enforcing nothing when the underlying timer is exposed.